src/commands/start.ts

```diff
@@ -134,13 +136,28 @@ export default class Start extends Command {
           }
         });
 
-        cmd.stderr.on('data', () => {
-          this.warn(_error(`Error executing architect-${service_config.language}-launcher`));
-          this.warn(_error(`Failed on: ${cmd_path} ${cmd_args.join(' ')}`));
-          this.exit(1);
+        let hadError = false;
+        readline.createInterface({
+          input: cmd.stderr,
+          terminal: false
+        }).on('line', data => {
+          hadError = true;
+          data = data.trim();
+          if (data.length > 0) {
+            if (service_config.isScript()) {
+              this.log(_error(data));
+            } else {
+              this.log(_info(`[${service_config.name}]`), _error(data));
+            }
+          }
         });
 
         cmd.on('close', () => {
+          if (hadError) {
+            this.error(_error(`Error executing architect-${service_config.language}-launcher`));
+            this.error(_error(`Failed on: ${cmd_path} ${cmd_args.join(' ')}`));
```

Comment:
The `this.error()` method in the oclif framework actually includes an exit clause. This prevents the second error from from ever executing and thus makes it so we can't tell which service was the culprit.